lab-teddy/app/.env

```diff
@@ -0,0 +1 @@
+API_URL=http://localhost:3000
```

Comment:
Got to make sure to gitignore this file in every project you use an env. Gotta take one point off for this.